dashboard/utils/serviceHelper.ts

```diff
@@ -0,0 +1,229 @@
+// Define a union type for supported cloud providers.
+export type Providers =
+  | 'aws'
+  | 'gcp'
+  | 'digitalocean'
+  | 'azure'
+  | 'civo'
+  | 'kubernetes'
+  | 'linode'
+  | 'tencent'
+  | 'oci'
+  | 'scaleway'
+  | 'mongodbatlas'
+  | 'ovh'
+  | 'scaleway'
+  | 'tencent';
+
+// Define an object that maps each provider to an array of its supported services.
+export const allProvidersServices: { [key in Providers | string]: string[] } = {
```

Comment:
```suggestion
export const allProvidersServices: { [key in Providers]: string[] } = {
```

`| string` part makes `key in Providers` redundant

Reply:
Oh forgot to change that